### ISSUE TITLE
#load does not issue error message when file does not exist on win 7

### ISSUE THREAD

**Issue by markek** [fsprojects/Fable#221]
### Description

#load does not issue error message when file does not exist on win 7

### Repro steps

#load "no such file"

### Expected behavior

Something like fsi:

stdin(4,1): error FS0078: Unable to find the file 'no such file' in any of ...

### Actual behavior
no error message

### Related information 

* win 7

**Comment by alfonsogarciacaro** [comment]
We can check that all the files referenced in a project or a script actually exist, but you already get an error if you're trying to call a method from a non-existent file. So I guess you're trying to use `#load` to _activate_ another file as it's usually done with `require` in node.

This can be problematic because `#load <file-path>` doesn't automatically translate to `require(<file-path>)`: specific imports are created instead for each member of the referenced file as needed (e.g., if the referenced file contains two functions `multiply` and `divide` and you just call the first one, only one `import {multiply} from <file-path>` will be generated).

This allows dead-code removal (tree shaking) when bundling your app but it also means that, if you don't actually call any function from the referenced file, no import will be generated.

In conclusion, we can add the file check, but I just want to be sure whether this is actually what you need for your use case. If you're indeed trying to activate a file with a `#load` directive without calling any member from it, we could raise a warning about that. But it'll be a bit complicated because right now there's no simple way to get the specific files loaded by each script (you can have a script that loads another script that loads other files, etc).

**Comment by colinear** [comment]
Actually I had reduced the failure case to the simplest possible - I'm not trying to #load for side effects. I had never used #load previously, and the f# manual is not clear that it is similar to c-style #include. I'm still not sure if it is. Trying to share code between browser and node sides of an app.

At some point in experimenting I think I had some function definitions then a #load that masked those with new definitions, so it compiled without error yet was intended to use the defs from the #load. Then I found that #load works best (or at all) with a module def at the top. After I figured it all out I ended up with ... blah blah ... fourth attempt at succinct post... giving up for now... no opinion yet... opening a new ticket for a different difference between fsi and fable that is more tractable.. see issue https://github.com/fsprojects/Fable/issues/225

**Comment by alfonsogarciacaro** [comment]
This is tricky because in fsi you can _indeed_ use `#load` just for side effects. It's only in Fable that this behavior is different to allow tree shaking as explained above. I've added a note in the docs about this, I hope it helps.

(BTW, thanks for reporting the other issue. It was a good point and should be fixed now. I'll publish a new version with the fix shortly.)

**Comment by alfonsogarciacaro** [comment]
Sorry, I'm closing the issue as it's a bit old. Please feel free to reopen it if you still have the same problem with the latest version.
